packages/mobile-top-nav/src/media-menu.js

```diff
@@ -129,6 +130,7 @@ class MediaMenu extends LitElement {
         <div class="menu-group">
           ${this.mediaMenuOptionsTemplate}
           <media-slider
+            selectedMenuOption=${this.selectedMenuOption}
```

Comment:
I think you want to prefix this with a dot to bind it to the property, ie:
`.selectedMenuOption=${this.selectedMenuOption}`

https://lit-element.polymer-project.org/guide/templates#bind-properties-to-templated-elements

Reply:
That's probably something in need of updating across the components, since these aren't HTML attributes that need to be exposed.